slothy/targets/arm_v7m/arch_v7m.py

```diff
@@ -2128,6 +2138,18 @@ def core(inst,t,log=None):
                 add_comments(inst.source_line.comments)
             ldr.source_line = ldr_src
 
+        ldrs_reordered = []
```

Comment:
```suggestion
        # In case the address register is also contained in the
        # register list, we need to overwrite the address register
        # in the last ldr
        ldrs_reordered = []
```